content/ko/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -0,0 +1,238 @@
+
+`ScheduleDaemonSetPods` 로 데몬셋 파드에 `.spec.nodeName` 용어 대신 
+`NodeAffinity` 용어를 추가해서 데몬셋 컨트롤러 대신 기본 
+스케줄러를 사용해서 데몬셋을 예약할 수 있다. 이후에 기본 
+스케줄러를 사용해서 대상 호스트에 파드를 바인딩 한다. 만약 데몬셋 파드에 
+이미 노드 선호도가 존재한다면 교체한다. 데몬셋 컨트롤러는 
+데몬셋 파드를 만들거나 수정할때만 이런 작업을 수행하며, 
+데몬셋의 `spec.template` 은 변경되지 않는다.
+
+```yaml
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+    - matchFields:
+      - key: metadata.name
+        operator: In
+        values:
+        - target-host-name
+```
+
+또한, 데몬셋 파드에 `node.kubernetes.io/unschedulable:NoSchedule` 이 톨러레이션으로 
+자동으로 추가된다. 기본 스케줄러는 데몬셋 파드를 
+스케줄링시 `unschedulable` 노드를 무시한다.
+
+
+### 테인트와 톨러레이션(Taints and Tolerations)
+
+데몬 파드는 
+[테인트와 톨러레이션](/docs/concepts/configuration/taint-and-toleration)을 존중하지만, 
+관련 기능기능에 따라 자동적으로 데몬셋 파드에 다음과 같이 
+톨러레이션을 추가한다.
+
+| 톨러레이션 키                            | 영향       | 버전    | 설명                                                         |
+| ---------------------------------------- | ---------- | ------- | ------------------------------------------------------------ |
+| `node.kubernetes.io/not-ready`           | NoExecute  | 1.13+    | 네트워크 파티션과 같은 노드 문제가 발생해도 데몬셋 파드는 축출되지 않는다.                   |
+| `node.kubernetes.io/unreachable`         | NoExecute  | 1.13+    | 네트워크 파티션과 같은 노드 문제가 발생해도 데몬셋 파드는 축출되지 않는다.                   |
+| `node.kubernetes.io/disk-pressure`       | NoSchedule | 1.8+    |                                                              |
+| `node.kubernetes.io/memory-pressure`     | NoSchedule | 1.8+    |                                                              |
+| `node.kubernetes.io/unschedulable`       | NoSchedule | 1.12+   | 데몬셋 파드는 기본 스케줄러에 의해 예약할수 없는 속성을 허용한다.                                                         |
+| `node.kubernetes.io/network-unavailable` | NoSchedule | 1.12+   | 호스트 네트워크를 사용하는 데몬셋 파드는 기본 스케줄러에 의해 이용할 수 없는 네트워크 속성을 허용한다.                    |
+
+
+
+
+## 데몬 파드와 통신
+
+데몬셋의 파드와 통신할 수 있는 몇 가지 패턴은 다음과 같다.
+
+- **푸시(Push)**: 데몬셋의 파드는 통계 데이터베이스와 같은 다른 서비스로 업데이트를 보내도록
+  구성되어있다. 그들은 클라이언트들을 가지지 않는다.
+- **노드IP와 알려진 포트**: 데몬셋의 파드는 `호스트 포트`를 사용할 수 있으며, 노드 IP를 통해 파드에 접근할 수 있다. 클라이언트는 노트 IP를 어떻게든지 알고 있으며, 관례에 따라 포트를 알고 있다.
+- **DNS**: 동일한 파드 셀렉터로 [헤드리스 서비스](/docs/concepts/services-networking/service/#headless-services)를 만들고, 
+  그 다음에 `엔드포인트` 리소스를 사용해서 데몬셋을 찾거나 DNS에서 여러 A레코드를
+  검색한다.
+- **서비스**: 동일한 파드 셀렉터로 서비스를 생성하고, 서비스를 사용해서
+  임의의 노드의 데몬에 도달한다(특정 노드에 도달할 방법이 없다).
+
+## 데몬셋 업데이트
+
+만약 노드 레이블이 변경되면, 데몬셋은 새로 일치하는 노드에 즉시 파드를 추가하고, 새로 
+일치하지 않는 노드에서 파드를 삭제한다.
+
+사용자는 데몬셋이 생성하는 파드는 수정할 수 있다. 그러나 파드는 모든 
+필드가 업데이트 되는 것을 허용하지 않는다. 또한 데몬셋 컨트롤러는 
+다음에 노드(동일한 이름으로)가 생성될 때 원본 템플릿을 사용한다.
+
+사용자는 데몬셋을 삭제할 수 있다. 만약 `kubectl` 에서 `--cascade=false` 를 명시하면 
+파드는 노드에 남게 된다. 이후에 동일한 셀렉터로 새 데몬셋을 생성하면, 
+새 데몬셋은 기존 파드를 채택한다. 만약 파드를 교체해야 하는 경우 데몬셋은 
+`updateStrategy` 에 따라 파드를 교체한다.
+
+사용자는 데몬셋에서 [롤링 업데이트를 수행](/docs/tasks/manage-daemon/update-daemon-set/) 할 수 있다.
+
+## 대몬셋의 대안
+
+### 초기화 스크립트
+
+분명히 노드에서 데몬 프로세스를 직접적으로 시작해서 실행할 수 있다
```

Comment:
It is certainly possible to run daemon processes by directly starting them on a node

```suggestion
데몬 프로세스를 직접 노드에서 시작해서 실행하는 것도 당연히 가능하다.
```